### DISCUSSION TITLE
Kafka Sink Header Support

### DISCUSSION THREAD

**Discussion by DKaos** [vectordotdev/vector#8183] [Q&A] [answered]
Does the Kafka Sink support writing headers? Can't seem to find the docs on it if it does...

**Comment by jszwedko** [comment, marked as answer]
Hey @DKaos . Not currently, it appears, but that is something that should be relatively straightforward to add if you wanted to open an issue.

**Reply by DKaos** [reply]
Thanks @jszwedko. Will do. Cheers!
